extensions/geode-modules-session-internal/src/main/java/org/apache/geode/modules/session/internal/common/ClientServerSessionCache.java

```diff
@@ -124,6 +125,9 @@ private void createOrRetrieveRegion() {
       LOG.debug("Created session region: " + this.sessionRegion);
     } else {
       LOG.debug("Retrieved session region: " + this.sessionRegion);
+
+      // Register interest in case users provide their own client cache region
+      sessionRegion.registerInterestForAllKeys(InterestResultPolicy.KEYS);
```

Comment:
We could registerInterest in a common place for both user and internally created session regions.

Reply:
I think you are right - but this follows the pattern in catlina/ClientSessionCache. 
Could do refactoring at another time.